generators/python-v2/ast/src/TypeInstantiation.ts

```diff
@@ -143,7 +143,11 @@ export class TypeInstantiation extends AstNode {
                 }
                 break;
             case "str":
-                writer.write(`"${this.internalType.value}"`);
+                if (this.internalType.value.includes("\n")) {
```

Comment:
Idk if it's necessarily true that a newline in the string means it should be triple quoted. the user may want to have `"A\\nstring"` and so the decision to use single line or triple quote might need to come from the consumer. And at least for our use case, I think we could have templating node templates and jinja block templates always proactively generate triple quoted strings in our opinionated default path

Reply:
Just saw the thread @amckinney was advising otherwise. I also checked python's `ast` library and it seems like it treats it as the same AST node as well:
<img width="416" alt="Screenshot 2024-11-17 at 9 24 18 AM" src="https://github.com/user-attachments/assets/e2f72079-f085-4af4-82fd-5fcc0ea73a45">

so ultimately could go either way here, but will just note that I think this could cause drift in some edge cases between what the user specifies and what we generate